src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-label/annotation-label.component.less

```diff
@@ -22,66 +33,67 @@
 			}
 		}
 
-	span.annotation-label {
-		height: 32px;
-		position: absolute;
-		font-size: 17px;
-		color: white;
-		text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
-		top: calc(~'100% + 20px');
-	}
-
-	form.annotation-label-edit {
-		background: @background-color-3;
-		width: 140px;
-		height: 120px;
-		position: absolute;
-		top: 100%;
-		z-index: 1;
-		display: flex;
-		flex-direction: column;
-		justify-content: center;
-		align-items: center;
-		padding: 5px 15px;
-		box-sizing: border-box;
-
-		mat-form-field.label-form-field {
-			width: 100%
+		span.annotation-label {
+			height: 32px;
+			position: absolute;
+			font-size: 17px;
+			color: white;
+			text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+			top: calc(~'100% + 20px');
 		}
 
-		mat-form-field.label-font-size {
-			width: 100%;
-		}
-	}
+		form.annotation-label-edit {
+			background: @background-color-3;
+			width: 140px;
+			height: 120px;
+			position: absolute;
+			top: 100%;
+			z-index: 1;
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+			align-items: center;
+			padding: 5px 15px;
+			box-sizing: border-box;
 
-	@white-inactive: rgba(255, 255, 255, 0.8);
+			mat-form-field.label-form-field {
+				width: 100%
+			}
 
-	.mat-form-field {
-		input {
-			caret-color: white;
-			color: white;
+			mat-form-field.label-font-size {
+				width: 100%;
+			}
 		}
 
-		/deep/ .mat-select-arrow {
-			color: white;
-		}
+		@white-inactive: rgba(255, 255, 255, 0.8);
 
-		/deep/ .mat-form-field-label, /deep/ .mat-select-value-text {
-			color: @white-inactive !important;
-		}
+		.mat-form-field {
+			input {
+				caret-color: white;
+				color: white;
+			}
 
-		/deep/ .mat-form-field-underline {
-			background-color: @white-inactive !important;
-		}
+			/deep/ .mat-select-arrow {
```

Comment:
check why the change

Reply:
It should be like this. It wasn't correct before